### PR TITLE
Add Migration tests for Activity logs

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+        logEntriesInDb.Count.Should().Be(1);
+    }
+
+    private static async Task InsertOldActivityLogEntry(ApplicationDbContext dbContext, Guid id)
+    {
+        var logEntryTable = dbContext.Model.FindEntityType(typeof(ActivityLogEntry))!.GetTableName();
+
+        var logEntryQuery =
+            $@"INSERT INTO ""{logEntryTable}"" (
```
Comment (TopSwagCode) [inline domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs:79]: why use raw SQL??

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        await applyMigration.Should().NotThrowAsync();
+
+        var logEntriesInDb = dbContext.ActivityLogs.ToList();
+
+        logEntriesInDb.Count.Should().Be(1);
+        logEntriesInDb.First().OtherOrganizationName.Should().Be(string.Empty);
+        logEntriesInDb.First().OtherOrganizationName.Should().Be(string.Empty);
+    }
+
+    [Fact]
```
Comment (martinhenningjensen) [inline domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs:55]: Remove

**Review by martinhenningjensen — CHANGES_REQUESTED** [review]

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs (hunk 2 above)
Comment (sahma19) [reply domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs]: Done: https://github.com/Energinet-DataHub/energy-origin/pull/794/commits/becf2f54797f6d17ef48e9a590b952f1c14be8f2

File: domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs (hunk 1 above)
Comment (sahma19) [reply domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/ActivityLogOtherOrgFieldsTests.cs:60]: Apparently we have to

**Review by martinhenningjensen — APPROVED** [review]

**Pull request merged**